datm/cime_config/testdefs/testlist_datm.xml

```diff
@@ -61,6 +61,42 @@
       <option name="wallclock"> 00:10:00 </option>
     </options>
   </test>
+  <test compset="HIST_DATM%CRUJRA2024_SLND_SICE_SOCN_SROF_SGLC_SWAV_SESP" grid="f10_f10_mg37" name="SMS_Ld5">
+    <machines>
+      <machine name="derecho" compiler="intel" category="aux_cdeps"/>
+      <machine name="betzy" compiler="intel" category="aux_cdeps_noresm"/>
+    </machines>
+    <options>
+      <option name="wallclock"> 00:10:00 </option>
```

Comment:
Add a comment for each of the new tests. To say what's it doing and why.

So something like...

``` xml
<option name="comment">Test historical CRUJRA-2024 forcing in a short smoke test just to make sure it works</option>
```